README.md

```diff
@@ -107,6 +105,67 @@ Current community packages are hosted on https://packagecloud.io/StackStorm. For
 - [RHEL7/CentOS7](https://docs.stackstorm.com/install/rhel7.html)
 - [RHEL6/CentOS6](https://docs.stackstorm.com/install/rhel6.html)
 
+## Adding Support For a New Distribution
+
+If you are adding support for a new distribution for which ``packagingbuild`` and ``packagingtest``
+images are not yet published to Docker Hub and you want to test the build pipeline locally, you
+need to update ``docker-compose.yml`` file to use locally built Docker images.
+
+For example:
```

Comment:
Please include references to files and high-level ideas abut what needs to be done, but not the code samples as it's gets outdated quickly and every distro could have their own specifics.

Reply:
It's not meant to be copied directly, just to give user some general idea on what needs to be done.

Reply:
Will take a look at it again tomorrow and re-think / re-word it if I can, worst case, I will just remove it.

My goal was to just add something which helped me and I thought it may perhaps or also add others.